wemake_python_styleguide/visitors/tokenize/comments.py

```diff
@@ -81,6 +83,29 @@ def _check_noqa(self, token: tokenize.TokenInfo) -> None:
             # We cannot pass the actual line here,
             # since it will be ignored due to `# noqa` comment:
             self.add_violation(WrongMagicCommentViolation(text=comment_text))
+            return
+        self._check_forbidden_noqa(excludes)
+
+    def _check_forbidden_noqa(self, noqa_excludes) -> None:
+        excludes_list = noqa_excludes.split(',')
+        excludes_list = [ex.strip() for ex in excludes_list]
+        forbidden_noqa = ''.join(self.options.forbidden_inline_ignore)
+        forbidden_noqa_list = forbidden_noqa.split(',')
+        for fn in forbidden_noqa_list:
+            fn = fn.strip()
```

Comment:
What is `fn`? I think we need a better name here

Reply:
Changed to `noqa_code`